lib/preexec.bash

```diff
@@ -20,10 +19,9 @@ function __bp_require_not_readonly() { :; }
 
 # Disable trap DEBUG on subshells - https://github.com/Bash-it/bash-it/pull/1040
 __bp_enable_subshells= # blank
-set +T
```

Comment:
why did you remove this?

Reply:
It's handled upstream now, which I believe I put in the commit message.

It alsö breaks BATS so needs to be handled properly, not just forced, and upstream uses the variable to control it.

Reply:
I see, in any case, we should note that in the commit message, and probably update the comment in line 21 (disable trap via upstream preexec)

Reply:
Yeah, I squashed that commit...I had a note but then lost it 😃 

I'll update the note and the comment 👍

Reply:
```suggestion
# For performance, testing, and to avoid unexpected behavior: disable DEBUG traps in subshells.
# See bash-it/bash-it#1040 and rcaloras/bash-preexec#26
: "${__bp_enable_subshells:=}" # blank
```

Reply:
Done and pushed